lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -226,6 +226,18 @@ sub register {
         });
 
     $app->helper(
+        limit_previous_link => sub {
```

Comment:
Ok, but before moving this again, could you tell me in which specific file you want it. Just to prevent misunderstanding again.